src/antares/model/study.py

```diff
@@ -367,6 +371,35 @@ def wait_job_completion(self, job: Job, time_out: int = 172800) -> None:
         """
         self._run_service.wait_job_completion(job, time_out)
 
+    def read_outputs(self) -> list[Output]:
+        """
+        Load outputs into current study
+
+        Returns: Output list
+        """
+        outputs = self._output_service.read_outputs()
+        self._outputs = {output.name: output for output in outputs}
+        return outputs
+
+    def get_outputs(self) -> dict[str, Output]:
+        """
+        Get outputs of current study
+
+        Returns: (output_id, Output) dictionary
+        """
+        return self._outputs.copy()
```

Comment:
Instead of doing a copy what we do is we return a MappingProxyType. This basically does the same thing but we should keep the code coherent (see `get_binding_constraints` method for instance)